plugins/modules/ios_banner.py

```diff
@@ -43,6 +43,11 @@
     - incoming
     - slip-ppp
     type: str
+  delimiting_char:
```

Comment:
rename to:
```suggestion
  multiline_delimiter:
```
ref: https://docs.ansible.com/ansible/latest/collections/cisco/ios/ios_config_module.html#parameter-multiline_delimiter

Reply:
@justjais so for the config it is a multiline delimiter and here is delimiting character, kept it same as in the IOS device itself, for less confusion on its use case